### PR TITLE
Improve condor logging for infererence client jobs 

### LINKED ISSUE #141: Remove notebook from `export` task directory

### PR DESCRIPTION
**Pull request by EthanMarx**
- Points to updated `Pycondor` branch that allows for arbitrary suffixes to be appended to the log, error, and output paths specified in condor submit files. This supports adding things like `$(ProcID)` to files for dynamic file naming. 

- Adds this functionality to the inference condor deployment so that log files for individual jobs can be inspected easier .
- Handles edge case for saving TimeSeries to h5py files in chunked form
- Removes notebook from `aframe/tasks/export`

Closes #141

### CODE REVIEW

File: projects/infer/infer/data.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(
         self.shifts = [int(i * self.sample_rate) for i in shifts]
         self.stride = int(self.sample_rate / inference_sampling_rate)
         self.step_size = self.stride * batch_size
-
+        print(self.stride, self.step_size, self.batch_size, self.remainder)
```
Comment (wbenoit26) [inline projects/infer/infer/data.py:75]: You've got two `print` statements in this file

File: projects/infer/infer/main.py [hunk 2]
```diff
@@ -45,6 +46,7 @@ def infer(
             f"Submitting inference request {i} for sequence {sequence.id}"
         )
 
+        print(x.shape, x_inj.shape)
```
Comment (wbenoit26) [inline projects/infer/infer/main.py:49]: `print` statement here

**Review by wbenoit26 — CHANGES_REQUESTED** [review]
Just a couple of `print`s to remove

**Comment by EthanMarx** [comment]
Re-added that pre-commit hook for automated print removal :)

**Review by wbenoit26 — APPROVED** [review]

**Pull request merged**